packages/next/src/server/dev/next-dev-server.ts

```diff
@@ -1355,8 +1355,16 @@ export default class DevServer extends Server {
         page: this.actualInstrumentationHookFile!,
         clientOnly: false,
       })
+      const isSrc = this.actualInstrumentationHookFile.includes('src')
```

Comment:
```suggestion
      const isSrc = /[\\/]src[\\/]instrumentation\.[^\\/]+$/.test(this.actualInstrumentationHookFile)
```

Maybe it's better as the project path can contain `"src"`.

Reply:
the path is relative to the project root so it's not needed I think